packages/graphql-toolbox/src/components/editor/Editor.tsx

```diff
@@ -54,6 +54,7 @@ export const Editor = (props: Props) => {
     const [variableValues, setVariableValues] = useState("");
     const [output, setOutput] = useState("");
     const refForQueryEditorMirror = useRef<EditorFromTextArea | null>(null);
+    const isShowRightPanel = settings.isShowDocsDrawer || settings.isShowSettingsDrawer;
```

Comment:
**nitpick not-blocking** I'm not sure about the name `is...`, I understand that this is a boolean to define whether or not should show the right panel, not whether or not is a thing. Maybe `showRightPanel` or `shouldShowRightPanel` is a bit more clear?